pkg/controller/operands/kubevirt_test.go

```diff
@@ -21,11 +21,21 @@ import (
 	kubevirtv1 "kubevirt.io/client-go/api/v1"
 	"os"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	"strings"
 
 	virtconfig "kubevirt.io/kubevirt/pkg/virt-config"
 )
 
 var _ = Describe("KubeVirt Operand", func() {
+
```

Comment:
Maybe it will be better to move this const block under `Context("Feature Gates", func() {` ?

Reply:
It reused in the CM tests